docs/src/languages/php/_index.md

```diff
@@ -304,17 +226,197 @@ markdownify=false
 
 {{< /codetabs >}}
 
-
-
 {{% config-reader %}}
 [`platformsh/config-reader` Composer library](https://github.com/platformsh/config-reader-php)
 {{% /config-reader%}}
 
-## Runtime configuration
+## PHP settings
+
+PHP can be used in two modes:
+
+- CGI (common gateway interface): the mode used in the `passthru`. CGI is a PHP interface to interact with web servers. PHP-FPM (FastCGI Process Manager) is commonly used to run the CGI mode.
```

Comment:
When you introduce abbreviations, it should be the full term followed by the abbreviation: common gateway interface (CGI)
https://developers.google.com/style/abbreviations?hl=en#spelling-out

---

By "what benefits come from specifying the mode?", I meant what reasons are there to focus this bullet point on CGI rather than on PHP-FPM. Because this list structure seems to make CGI a strong focus when I'm not sure it should be interesting for users.